hw1/02-palindrome.html

```diff
@@ -6,20 +6,33 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body class="bg-primary">
     <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
+      <section class="d-flex justify-content-center vh-60">
+        <div class="bg-light border rounded p-4 my-5 w-50">
+          <h1 class="text-center">Exercise 02 - Palindrome</h1>
```

Comment:
If you carefully observe the desired output the title is not centered. You can remove the text-center class from the h1 tag.

Reply:
Implemented